contracts/engine/V3/forks/PROHIBITION/AdminACLV0_PROHIBITION.sol

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import "./interfaces/IAdminACLV0_PROHIBITION.sol";
+import "../../../../interfaces/0.8.x/IGenArt721CoreContractV3_Base.sol";
+
+import "@openzeppelin-4.7/contracts/access/Ownable.sol";
+import "@openzeppelin-4.7/contracts/utils/introspection/ERC165.sol";
+
+/**
+ * @title Admin ACL contract, V0.
+ * @author Art Blocks Inc.
+ * @notice Privileged Roles and Ownership:
+ * This contract has a single superAdmin that passes all ACL checks. All checks
+ * for any other address will return false.
+ * The superAdmin can be changed by the current superAdmin.
+ * Care must be taken to ensure that the admin ACL contract is secure behind a
+ * multi-sig or other secure access control mechanism.
+ */
+contract AdminACLV0_PROHIBITION is IAdminACLV0_PROHIBITION, ERC165 {
+    string public AdminACLType = "AdminACLV0";
```

Comment:
Suggestion to please make this a custom named type (e.g. `"AdminACLV0_PROHIBITION"`)

Reply:
Ah! I meant to do that too. Thanks. for catching this.

Should I update the `coreType` in the corev3 contract as well?

Reply:
> Should I update the coreType in the corev3 contract as well?

I don't believe so, though will defer to @ryley-o on this one.

Reply:
leave `coreType` unchanged (`GenArt721CoreV3_Engine_Flex`) 👍 That will let our indexing system treat it as an Engine Flex contract, which is what we want in this case 😃